release_images.yml

```diff
@@ -2,23 +2,23 @@
 release_images:
 # Check .release_images_template.yml for reference
   1:
-    framework: "tensorflow"
-    version: "2.8.0"
+    framework: "autogluon"
+    version: "0.3.2"
     arch_type: "x86"
     customer_type: "sagemaker"
     training:
       device_types: ["cpu", "gpu"]
-      python_versions: ["py39"]
-      os_version: "ubuntu20.04"
+      python_versions: ["py38"]
+      os_version: "ubuntu18.04"
```

Comment:
Is the OS version different for training and inference images?

Reply:
It is, yes. The [training CPU](https://github.com/aws/deep-learning-containers/blob/master/autogluon/training/docker/0.3.2/py3/Dockerfile.cpu) and [training GPU](https://github.com/aws/deep-learning-containers/blob/master/autogluon/training/docker/0.3.2/py3/cu112/Dockerfile.gpu) dockerfiles are based on ubuntu or nvidia docker images, which we're keeping close to the versions used by 0.3.1 dockerfiles, while the [inference CPU](https://github.com/aws/deep-learning-containers/blob/master/autogluon/inference/docker/0.3.2/py3/Dockerfile.cpu) dockerfile uses MX 1.9 with UL20 as base, now, instead of MX 1.8 with UL18 on the 0.3.1 dockerfile.